### ISSUE TITLE
Tbomb not working in Nigeria

### ISSUE THREAD

**Issue by fadzaktech** [TheSpeedX/TBomb#201]
Why doesn't it work in Nigeria?

**Comment by 0n1cOn3** [comment]
Missing API‘s.

➖
Ultragate-Solutions
Stefan Kuny | Inhaber & Gründer
Im Gässli 1
4228 Erschwil

+41763289408
Info@ultragate-solutions.ch
https://ultragate-solutions.ch


Confidentiality Note: This message is intended only for the use of the named recipient(s) and may contain confidential and/or privileged information. If you are not the intended recipient, please contact the sender and delete this message.
Any unauthorized use of the information contained in this message is prohibited.
________________________________
Von: fadzaktech <notifications@github.com>
Gesendet: Thursday, October 29, 2020 8:36:56 AM
An: TheSpeedX/TBomb <TBomb@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: [TheSpeedX/TBomb] Tbomb not working in Nigeria (#201)


Why doesn't it work in Nigeria?

**Comment by 0n1cOn3** [comment]
Missing API

**Comment by CPT-Damian** [comment]
Worked once yesterday, I'm still testing it. I'll keep trying to see.

**Comment by 0n1cOn3** [comment]
Hey @CPT-Damian 

Please join our Support and Info Channel on Telegram. We need further details about your results to detect the issue.

https://t.me/TBombChat

**Comment by CPT-Damian** [comment]
It does now. Not just call spam.

**Comment by 0n1cOn3** [comment]
> @0n1cOn3 the issue is not yet solved it does not work in Nigeria.

Yes, we dont have any data for Nigeria where we can use.. But we still looking for extended the options.

**Comment by TheSpeedX** [comment]
Hello!

After careful consideration, I’ve decided to close all open issues for this repository. 
This project has grown into something beyond my initial expectations, but I’ve found it increasingly difficult to maintain due to low-quality contributions and spam pull requests/issues. Furthermore, I am transitioning to new ventures, and I won’t be able to continue contributing to TBomb any further.

Though I’m stepping away, I’m grateful to all those who have used and supported this tool. TBomb will remain usable through Git and PyPI, but I won’t be actively involved moving forward.

Thank you for your understanding and for using this tool!
Best regards,
TheSpeedX
